### ISSUE TITLE
Support for Route Groups.

### ISSUE THREAD

**Issue by i-tengfei** [apteryxxyz/next-ws#2]
"next-ws is a really great library and I'm using it to implement Apollo Server subscriptions. However, when I use it within [Route Groups](https://nextjs.org/docs/app/building-your-application/routing/route-groups), I encounter the following error:"

`/app/(api)/graphql/route.ts`

```
- ready [next-ws] websocket server started successfully
- error unhandledRejection: PageNotFoundError: Cannot find module for page: /graphql/route
    at findPagePathData (/node_modules/next/dist/server/dev/on-demand-entry-handler.js:304:15)
    at async Object.ensurePage (/node_modules/next/dist/server/dev/on-demand-entry-handler.js:476:38)
    at async Server.<anonymous> (/node_modules/next-ws/server/hook.js:37:9) {
  code: 'ENOENT'
}
```

**Comment by apteryxxyz** [comment]
👋 I'll look into, do some testing and see if I can get route groups to work.

**Comment by apteryxxyz** [comment]
Good news, I just found a fix for this issue, I'll implement it, test it, then publish.

**Comment by apteryxxyz** [comment]
I've just published `next-ws@0.2.2`, which should now fix this issue.
